hack/quickstart/install.sh

```diff
@@ -18,6 +20,16 @@ helm install argocd argo-cd \
   --set notifications.enabled=false \
   --set server.service.type=NodePort \
   --set server.service.nodePortHttp=31443 \
+  --set server.extensions.enabled=true \
+  --set 'server.extensions.contents[0].name=argo-rollouts' \
+  --set 'server.extensions.contents[0].url=https://github.com/argoproj-labs/rollout-extension/releases/download/v0.3.3/extension.tar' \
+  --wait
+
+helm install rollouts argo-rollouts \
+  --repo https://argoproj.github.io/argo-helm \
+  --version 2.32.8 \
+  --create-namespace \
+  --namespace rollouts \
```

Comment:
How about adding `argo` prefix for the consistency (with `argocd`)?
```suggestion
  --namespace argo-rollouts \
```

Reply:
`argo-rollouts` it the commonly installation namespace for argo-rollouts so agree with @devholic